src/shared/constants/urls.ts

```diff
@@ -43,6 +43,16 @@ export const websiteSections: Partial<Record<keyof typeof website, string>> = {
     cart: 'кошик',
 };
 
+export const NAV_LINKS = {
+    main: { href: '/', label: websiteSections.main },
+    categories: { href: '/categories', label: websiteSections.categories },
+    services: { href: '/services', label: websiteSections.services },
+    about: { href: '/about', label: websiteSections.about },
+    reviews: { href: '/reviews', label: websiteSections.reviews },
+    cart: { href: '/cart', label: websiteSections.cart },
+    profile: { href: '/profile', label: websiteSections.profile },
+} as const;
+
```

Comment:
Тут my bad. Вчора з Олександром вирішили, що websiteSections це оверхед, тому пропоную замість NAV_LINKS щось таке:

 ```suggestion
export const websiteSections = [
    { label: 'головна', link: website.main },
    { label: 'категорії', link: website.categories },
    { label: 'послуги', link: website.services },
    { label: 'про нас', link: website.about },
    { label: 'відгуки', link: website.reviews },
    { label: 'часті питання', link: website.faq },
    { label: 'особистий кабінет', link: website.profile },
    { label: 'кошик', link: website.cart },
];

```